plugins/modules/provision_workflow_manager.py

```diff
@@ -911,30 +908,37 @@ class instance for further use.
             self.log("The provisioned status of the wired device is {0}".format(status), "INFO")
 
             if status == "success":
-                try:
-                    response = self.dnac_apply['exec'](
-                        family="sda",
-                        function="re_provision_wired_device",
-                        op_modifies=True,
-                        params=self.want["prov_params"],
-                    )
-                    self.log("Reprovisioning response collected from 're_provision_wired_device' API is: {0}".format(response), "DEBUG")
-                    task_id = response.get("taskId")
-                    self.get_task_status(task_id=task_id)
-                    self.result["changed"] = True
-                    self.result['msg'] = "Re-Provision done Successfully"
-                    self.result['diff'] = self.validated_config
-                    self.result['response'] = task_id
-                    self.log(self.result['msg'], "INFO")
-                    return self
+                if self.validated_config.get("provisioning") is True:
```

Comment:
Why do we have to re-provision a wired device if it's already provisioned? Can we use a variable to do it? If force_provisioning is set to true, then re-provision the device even if it's already provisioned. Otherwise, do nothing. By default, set this to false. 
This is only applicable to wired devices as of now. Please document this clearly

Reply:
added the new feature